dev/io.openliberty.microprofile.openapi.2.0.internal_fat/publish/servers/OpenAPIMergeTestServer/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.ws.beta.edition=true
```

Comment:
why not put this in bootstrap.properties?

Reply:
It doesn't work in bootstrap.properties because the server reads the system property and stores the value before it reads bootstrap.properties. (That took ages to work out. I did update the wiki page but didn't think to add a comment here.)